source/qml/Library/RendererQml/NumberInputRender.cpp

```diff
@@ -39,21 +39,21 @@ std::shared_ptr<RendererQml::QmlTag> NumberinputElement::getQmlTag()
 
 void NumberinputElement::initialize()
 {
+    const std::string origionalElementId = mInput->GetId();
+    mInput->SetId(mContext->ConvertToValidId(mInput->GetId()));
+    mOrigionalElementId = mInput->GetId();
```

Comment:
There are still other places we are declaring the variable for ex in contentItemTag function(). Please remove those

Reply:
Resolved in the latest commit